metronome/hotstuff/service/src/io/iohk/metronome/hotstuff/service/ConsensusService.scala

```diff
@@ -34,20 +35,26 @@ import scala.util.control.NonFatal
   *
   * It handles the `consensus.basic.Message` events coming from the network.
   */
-class ConsensusService[F[_]: Timer: Concurrent, N, A <: Agreement: Block](
+class ConsensusService[F[
+    _
+]: Timer: Concurrent, N, A <: Agreement: Block: Signing](
```

Comment:
This is the first time I see scalafmt fail so spectacularly. Fortunately it can be prettified to something like:

```scala
class ConsensusService[
    F[_]: Timer: Concurrent,
    N,
    A <: Agreement: Block: Signing
](
```

Reply:
Wow! I tried many times but without the final linebreak before the `]` it just put it back the way it was. Thanks!